demisto_sdk/commands/validate/file_validator.py

```diff
@@ -776,18 +788,32 @@ def validate_all_files(self, skip_conf_json):
                 for file_name in os.listdir(dir_path):
                     file_path = os.path.join(dir_path, file_name)
 
-                    is_yml_file = file_name.endswith('.yml') and \
-                        dir_name in (constants.INTEGRATIONS_DIR, constants.SCRIPTS_DIR)
+                    if os.path.isfile(file_path):
```

Comment:
there is another function like this, let's fix her too

Reply:
done - validate_all_files_schema